schemas/hook-status.yml

```diff
@@ -0,0 +1,55 @@
+$schema: http://json-schema.org/draft-04/schema#
+title:   "Hook status response"
+description: |
+  A snapshot of the current status of a hook.
+type: object
+properties:
+  lastFire:
```

Comment:
Is `lastFire` supposed to be optional? Maybe the initial value can just be an error message saying it haven't been triggered yet..